apps/DesktopStreamer/CMakeLists.txt

```diff
@@ -38,8 +38,8 @@ if(APPLE)
   else()
     list(APPEND DESKTOPSTREAMER_LINK_LIBRARIES "-framework CoreGraphics")
   endif()
-elseif(WIN32)
-  list(APPEND DESKTOPSTREAMER_LINK_LIBRARIES Ws2_32)
+elseif(MSVC)
+  list(APPEND DESKTOPSTREAMER_LINK_LIBRARIES Ws2_32 Qt5::WinMain)
```

Comment:
Yes, it has a missing link to WinMain. Probably related to Qt5 port, so I would guess it's only needed for qt_support. Console applications have a regular main, so it's not needed there.